shared/chat/conversation/messages/text/container.js

```diff
@@ -11,10 +11,10 @@ import type {OwnProps} from './container'
 
 const getProps = createCachedSelector(
   [Constants.getMessageFromMessageKey, Constants.getEditingMessage],
-  (message: Constants.TextMessage, editingMessage) => ({
+  (message: ?Constants.TextMessage, editingMessage) => ({
     isEditing: message === editingMessage,
-    text: message.message.stringValue(),
-    type: message.messageState,
+    text: message ? message.message.stringValue() : null,
```

Comment:
its technically possible the store won't have a value here, happens in mock but never in production hopefully